packages/api-gateway/__tests__/geolocation/getPlace.test.js

```diff
@@ -0,0 +1,126 @@
+const Fastify = require('fastify');
+const Autoload = require('fastify-autoload');
+const pluginLoader = require('fastify-plugin');
+const boom = require('@hapi/boom');
+const getApiGatewayDirectory = require('../../src/utils/getApiGatewayDirectory');
+
+describe('Get a place then return a object and status code 200', () => {
+  // mock update place
+  const getPlace = () => async (id) => ({
+    id,
+    location: [-17.78, 78.878],
+    country: 'Saint Lucia',
+    state: 'Colorado',
+    city: 'North Adonisside',
+    zipcode: '18489-4272',
+    street_address: '17010 Lamar Lake',
+    place_db_id: '483bbd21-ca8f-4be6-ae51-0d291408d165',
+    created_at: '2022-04-27T07:24:51.097Z',
+    updated_at: '2022-04-27T23:47:07.252Z',
+    deleted_at: '2022-04-27T23:47:07.252Z',
```

Comment:
@Edilberto-Vazquez what do you think if instead of using hardcoded values, we use the [Faker](https://fakerjs.dev/guide/) library in order to generate this information?